core/bridges/bridge_type.go

```diff
@@ -140,61 +140,61 @@ func MarshalBridgeMetaData(latestAnswer *big.Int, updatedAt *big.Int) (map[strin
 	return mp, nil
 }
 
-// TaskType defines what Adapter a TaskSpec will use.
-type TaskType string
+// BridgeName defines what Adapter a TaskSpec will use.
+type BridgeName string
```

Comment:
TaskType was a left over name from the v1 pipeline from what I can tell. This is only used for bridge names so just renaming for clarity